config/application.rb

```diff
@@ -17,6 +18,9 @@
 
 module Mmt
   class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 5.1
```

Comment:
This setting controls the same things as the initializers 'new_framework_defaults...'.  In my mind, one of two things is happening:
1. This setting is overriding those initializers and those files can be deleted.
2. This setting is being overridden by the initializer for 5.1 which is currently completely false.

I suspect it is the former, but we should understand which is happening.

Reply:
1) is true.  However,  the initializer files were left around so we can refer to them as needed.  The intent was to leave them in-place until we are completely finished with Rails upgrade. At this point, they should probably be removed after the 6.0 upgrade, or after we decide to postpone the 6.0 upgrade.